recipe-server/normandy/settings.py

```diff
@@ -358,6 +363,9 @@ def RAVEN_CONFIG(self):
 
     GITHUB_URL = values.Value('https://github.com/mozilla/normandy')
 
+    # URL that hosts Experimenter experiment definitions
+    EXPERIMENTER_HOST = values.URLValue('https://experimenter.services.mozilla.com')
```

Comment:
This still needs docs.

Reply:
This probably also shouldn't have a default value. I doubt we want to use prod experimenter in the stage environment of Normandy.